libs/interfaces/evaluation/evaluation.interface.ts

```diff
@@ -1,10 +1,12 @@
 import {IEvaluationTag} from '..';
+import {IGroup} from '../group/group.interface';
 
 export interface IEvaluation {
   id: string;
   filename: string;
   readonly data?: Record<string, any>;
   evaluationTags: IEvaluationTag[];
+  groups?: IGroup[];
```

Comment:
Are there any instances left where we don't return the groups with evaluations? I'm just wondering if we can make this no longer optional.